templates/security/disclosure-policy.html

```diff
@@ -0,0 +1,124 @@
+{% extends "templates/one-column.html" %}
+
+{% block title %}Ubuntu Security disclosure and embargo policy{% endblock %}
+
+{% block meta_description %}Canonical and the Ubuntu Security Team participate in responsible disclosure and collaborate with the wider community on security issues. This describes how to contact the Ubuntu Security Team, what you can expect when you contact us, and what we expect from you.{% endblock %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1WdamniXOHemuPzl6vESYD9StPgh4-PepbqdVHYrzbuI/edit{% endblock meta_copydoc %}
+
+{% block content %}
+<section class="p-strip--suru-topped is-bordered">
+  <div class="row u-equal-height">
+    <div class="col-8 u-vertically-center">
+      <h1>
+        Ubuntu Security disclosure and embargo policy
+      </h1>
+      <p>
+        <i>Valid since: Oct-2020</i>
```

Comment:
```suggestion
        <i>Valid since: October 2020</i>
```
This is our Month Year format